boa_engine/src/lib.rs

```diff
@@ -19,7 +21,12 @@
 //! ```
 //! use boa_engine::{Context, Source};
 //!
-//! let js_code = "console.log('Hello World from a JS code string!')";
+//! let js_code = r#"
```

Comment:
To make sure that we don't miss this again, I would add, in line 45, a panic (hidden with `#`), so that we catch the error.

Reply:
Updated 😄